src/Shared/ServerSentEvents/README.md

```diff
@@ -0,0 +1,11 @@
+# System.Net.ServerSentEvents
+
+Polyfill for the System.Net.ServerSentEvents library, including the `SseFormatter` component available in .NET 10.
```

Comment:
Do we need `#if NET_10_OR_GREATER` in the relevant files then?

Reply:
Given that it's not bundled in the shared framework, it seems orthogonal? If and when we add a dependency to the 10.0.0 version of the package, it would be added for all targets and the polyfill can be removed.